drivers/hmis_supplemental/app/views/hmis_supplemental/data_sets/_form.haml

```diff
@@ -0,0 +1,24 @@
+- url = data_set.persisted? ? url_for(id: data_set.id, action: :update) : url_for(action: :create)
+
+= simple_form_for data_set, url: url, as: :data_set do |f|
+  - if data_set.errors.any?
+    .alert.alert-danger{style: 'display: block;'}
+      %p Failed to save
+      %ul.error_list
+        - data_set.errors.full_messages.uniq.each do |msg|
+          %li= msg
+
+  .form-inputs
+    = f.input :name, required: true
+    = f.input :owner_type, collection: ['enrollment', 'client'], required: true, as: :radio_buttons
+    = f.input :field_config, as: :text, label: 'Field Configuration JSON', hint: 'example: `[{"key":"widget","type":"float","label":"The Widget","multiValued":false}]`', input_html: {rows: 10}
+    = f.fields_for(:remote_credential) do |sf|
+      %fieldset
+        %legend S3 Configuration
+        = sf.input :region, as: :string, label: 'S3 region'
```

Comment:
This should be a collection
```suggestion
        = sf.input :region, label: 'S3 region', collection: GrdaWarehouse::RecurringHmisExport.available_s3_regions, as: :select_two
```
Hmm, that list should probably be moved to `AwsS3` though.

Reply:
sure I'll change it and move that constant to awss3.